interface/views.py

```diff
@@ -32,7 +32,7 @@ def login_view(request):
         if form.is_valid():
             user = authenticate(username=form.data['username'],
                                 password=form.data['password'])
-            if user and user.username in settings.ACS_USER_WHITELIST:
```

Comment:
Q: @Jokeswar shouldn't we remove the ```ACS_USER_WHITELIST``` from settings?

Reply:
I removed it from settings since it's not used anywhere else (checked with @Jokeswar)